regress.sh

```diff
@@ -1,18 +1,25 @@
+# upgrade pip
+python -m pip install --upgrade pip
+
 # install HardFloat
-wget -O install_hardfloat.sh https://git.io/JJ5YF
+curl -L https://git.io/JJ5YF > install_hardfloat.sh
```

Comment:
Although I wasn't able to get the regression flow for Windows running for this project, I did try to fix as many things as possible to support Windows testing in the future.  One of those items is using ``curl`` rather than ``wget`` in regression scripts, since ``curl`` is available by default in a **bash** shell on a GitHub Actions Windows machine.